yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -399,7 +402,7 @@ func createMigrationAssessmentCompletedEvent() *cp.MigrationAssessmentCompletedE
 			TotalColocatedSize: sizeDetails.TotalColocatedSize,
 			TotalShardedSize:   sizeDetails.TotalShardedSize,
 		},
-		MigrationComplexity: getMigrationComplexity(), //TODO: to figure out proper algorithm
+		MigrationComplexity: schemaAnalysisReport.SchemaSummary.MigrationComplexity,
```

Comment:
minor suggestion, see if makes the code simpler: you can still call getMigrationComplexity() here and wherever it is required, and in that function have a check like if schemaSummary.MigrationComplexity != "" then return